packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -26,9 +26,12 @@ spec:
           volumeMounts:
             - mountPath: /fluent-bit/etc/
               name: config
-            - mountPath: /run/log/journal
+            - mountPath: /var/log/journal
               name: journal
               readOnly: true
+            - mountPath: /etc/machine-id
```

Comment:
We have not added a policy for this in SELinux, I do not think it will pass. But we can decide if that is dealbreaker for 2.5.8, or if we can fix quickly. Looks like we didn't have one for `/run/log` either, but thats ok now.

Reply:
> We have not added a policy for this in SELinux, I do not think it will pass

Correct, /etc/machine-id has its own label that's not part of our policy yet:

```
$ sudo semanage fcontext -l | grep /etc/machine-id
/etc/machine-id                                    regular file       system_u:object_r:machineid_t:s0
```

> Looks like we didn't have one for /run/log either

We do have that:

```
$ sudo ls -Zd /run/log/journal
drwxr-sr-x. root systemd-journal system_u:object_r:syslogd_var_run_t:s0 /run/log/journal
```

https://github.com/rancher/rancher-selinux/blob/main/policy/centos7/rancher.te#L27-L28

Reply:
@cmurphy can you work on a PR to add the machine-id policy to rancher selinux, this may not get tested till next monday so its a little lower priority.

Reply:
It actually looks like it doesn't need an additional policy, I was able to run a pod that mounts and reads /etc/machine-id on an selinux-enforcing host with no permission errors.

Reply:
Hmm, I wonder if that exemption is in vanilla SELinux or somewhere else ? I'm guessing SELinux if this was a basic setup.

Reply:
I think it's just default in plain selinux

```
$ sudo sesearch --allow --class file | grep machineid_t
   allow init_t machineid_t : file { ioctl read write create getattr setattr lock append unlink link rename mounton open } ; 
   allow domain machineid_t : file { ioctl read getattr lock open } ;
```

It's not a protected file, it's not secret information. The journald logs are different because those have sensitive data.